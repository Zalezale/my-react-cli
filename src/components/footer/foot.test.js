import React from 'react'
import Footer from './index.js'
import renderer from 'react-test-renderer'

test('Link changes the class when hovered', () => {
    const component = renderer.create(
        <Footer />,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})