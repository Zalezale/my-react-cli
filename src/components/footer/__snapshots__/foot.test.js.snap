// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="footer"
  style={
    Object {
      "background": "yellow",
    }
  }
>
  <h5>
    我是底部哦
  </h5>
</div>
`;
